packages/common/src/routes.ts

```diff
@@ -0,0 +1,8 @@
+import { Address } from "wagmi";
+import { ChainId } from "./chains";
+
+export const Explorer = {
```

Comment:
Seeing `Explorer` at calling site, I'd automatically think "class" or "type" because that's what I see capitalized names most used for.

Reply:
hmm true, I might just split the app routes in their own files, so:

```ts
import { applicationsPath } from "./routes/explorer";
```